### PR TITLE
Add Legacy blast

### PR DESCRIPTION
**Pull request by colinbrislawn**
Addresses #10. Comments welcome!

### CODE REVIEW

**Comment by colinbrislawn** [comment]
The Windows version is distributed in an executable archive. So I have to have a way to run it / extract it from the command line in `build.bat`. I'll look into this in anticipation of #13.

**Comment by colinbrislawn** [comment]
Extracting from .exe files from linux terminal. http://superuser.com/a/737244
Extracting from .exe files from Windows CLI: https://support.microsoft.com/en-us/kb/912203

**Comment by colinbrislawn** [comment]
This builds fine on Linux. The OSX version is failing strangely...
https://travis-ci.org/biocore/conda-recipes/jobs/90787082

Suggestions?

**Comment by colinbrislawn** [comment]
Changed name to `blast-legacy` to address #38

**Comment by colinbrislawn** [comment]
@Jorge-C, any ideas why this may be failing only on OSX? I'm at a loss.

**Comment by Jorge-C** [comment]
After inspecting the log, it may be a conda bug? I [opened an issue](https://github.com/conda/conda-build/issues/675) in their repo.

```
Fixing permissions
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:
    https://github.com/conda/conda-build/issues
Include the output of the command 'conda info' in your report.
Traceback (most recent call last):
  File "/Users/travis/miniconda/bin/conda-build", line 5, in <module>
    sys.exit(main())
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/main_build.py", line 190, in main
    args_func(args, p)
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/main_build.py", line 468, in args_func
    args.func(args, p)
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/main_build.py", line 391, in execute
    override_channels=args.override_channels, include_recipe=args.include_recipe)
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/build.py", line 466, in build
    post_build(m, sorted(files2 - files1))
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/post.py", line 355, in post_build
    mk_relative(m, f)
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/post.py", line 324, in mk_relative
    mk_relative_osx(path)
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/post.py", line 244, in mk_relative_osx
    s = macho.install_name_change(path, osx_ch_link)
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/macho.py", line 88, in install_name_change
    for link in otool(path):
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/macho.py", line 62, in otool
    assert line[0] == '\t', path
AssertionError: /Users/travis/miniconda/envs/_build/bin/bl2seq
Traceback (most recent call last):
  File "ci/main.py", line 138, in <module>
    build_upload_recipes(sys.argv[1], sys.argv[2])
  File "ci/main.py", line 45, in build_upload_recipes
    build(root)
  File "ci/main.py", line 67, in build
    check_call(build_cmd, shell=True)
  File "/Users/travis/miniconda/lib/python3.5/subprocess.py", line 584, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'conda build "recipes/legacyblast"' returned non-zero exit status 1

The command "python ci/main.py recipes biocore" exited with 1.
```

**Comment by colinbrislawn** [comment]
Thanks! :smile:

**Comment by colinbrislawn** [comment]
Build works and commits have been squashed. @Jorge-C @RNAer, merge me!

File: recipes/blastlegacy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  url: ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/2.2.22/blast-2.2.22-universal-macosx.tar.gz   # [osx]
+  fn: blast-2.2.22-x64-win64.exe                                                                          # [win64]
+  url: ftp://ftp.ncbi.nlm.nih.gov/blast/executables/release/2.2.22/blast-2.2.22-x64-win64.exe             # [win64]
+
+build:
+  number: 0
+  script:
+    - mkdir -p "$PREFIX/bin"                                  # [osx or linux64]
+    - mv -v $SRC_DIR/bin/* $PREFIX/bin                        # [linux64]
+    - ditto --rsrc -v --arch i386 $SRC_DIR/bin/* $PREFIX/bin  # [osx]
```
Comment (Jorge-C) [inline recipes/blastlegacy/meta.yaml:21]: Can you please add a comment mentioning the conda issue that this works around? Thanks!

**Comment by Jorge-C** [comment]
Minor request before merging :)

**Comment by RNAer** [comment]
:+1:

**Comment by colinbrislawn** [comment]
Done!

**Comment by Jorge-C** [comment]
Thank you! Happy Thanksgiving!!

**Comment by colinbrislawn** [comment]
Thanks. Happy Thanksgiving! :chicken: :sad_no_turkey_emoji:

**Pull request merged**